resdk/tests/functional/duplication/e2e_duplication.py

```diff
@@ -0,0 +1,20 @@
+# pylint: disable=missing-docstring
+from resdk.tests.functional.base import BaseResdkFunctionalTest
+
+
+class TestDuplication(BaseResdkFunctionalTest):
+
+    def test_collection_duplication(self):
```

Comment:
@dblenkus travis-ci doesn't seem to [run these tests](https://travis-ci.org/genialis/resolwe-bio-py/jobs/565663036#L406-L413). Is this expected?

Reply:
I think I found the problem. There no `__init__.py` file in the directory.

Reply:
Though is seems that not one functional test is actually tested.

Reply:
Oh, never mind, I think Jenkins runs them.

Reply:
Yes, e2e tests are run by Jenkins.